src/lib/mina_graphql/itn_zkapps.ml

```diff
@@ -0,0 +1,293 @@
+open Core
```

Comment:
Why `itn_zkapps`? How are they different to usual zkapps?

Reply:
In `zkapps` module regular logic to send a zkapp command is kept. In `itn_zkapps` there is a code for scheduler of a series of zkapp commands designed for testing on ITN. This is quite different and more complex, also not to be used in production

Reply:
If it's not going to be used in production, then I wonder why it is part of the repo? Why does the daemon need to be specially augmented for ITN testing? This feels wrong. After all, tests should test the production code, not some other code written specifically to support testing… Anyway, I guess this has been written previously and you're just moving it around and adapting, so I won't object to that. Still, I'd like to learn the answers to the questions above.

Reply:
It's used for testnet testing, for ITN specifically. When I say it's not for production, I mean that it's not to be used on mainnet. We cut ITN-related functionality off mainnet builds by using compile-time configuration.

Reply:
But that's my point exactly! Effectively this means ITN is testing a different node than the one we're going to release for mainnet. What do I misunderstand here?

Reply:
That's certainly true. That's why we try to include as little as possible into this ITN toolkit and be transparent about that. Having this functionality in mainnet is a bad idea, but not having it in test network reduces our ability to test.

ITN toolkit is a compromise in the middle.

Reply:
Understood. Thanks for explanation!